src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -64,7 +64,7 @@
                 </div>
               </template>
               <template #customAdvanced2>
-                <chi-date-picker id="date-picker" @chiDateChange="e => dateChangeHandler(e)" />
+                <chi-date-picker @chiDateChange="e => dateChangeHandler(e)" />
```

Comment:
Actually, this and all the other examples with no label probably throws a Missing label accessibility error. Check it, please

Reply:
Indeed, there are multiple errors like that :S